models/prices/optimism/prices_optimism_tokens_curated.sql

```diff
@@ -1,119 +0,0 @@
-{{ config(
-        schema='prices_optimism',
-        alias = alias('tokens_curated'),
-        materialized='table',
-        file_format = 'delta',
-        tags=['static']
-        )
-}}
-SELECT
-    TRIM(token_id) as token_id
-    , 'optimism' as blockchain
-    , TRIM(symbol) as symbol
-    , LOWER(TRIM(contract_address)) as contract_address
-    , decimals
-FROM
-(
-    VALUES
-
-    --tokens not yet supported or are not active on coinpaprika are commented out
-    ("op-optimism", "OP", "0x4200000000000000000000000000000000000042", 18),
```

Comment:
Tried to wrap my head around this but not sure how optimism prices work. but it looks like this table should be included in the tokens list?